services/docker-compose.local.yml

```diff
@@ -85,6 +85,18 @@ services:
       - "8080"
       - "3011:3000"
```

Comment:
Q: what is going to happen in my local folder /var/lib/docker/volumes?
when I run make up-prod or up-devel?